.github/workflows/static.yml

```diff
@@ -12,10 +12,6 @@ jobs:
       - name: Checkout code
         uses: actions/checkout@v2
 
-      - name: Download dependencies
-        run: |
-          composer update --no-interaction --prefer-dist --optimize-autoloader
```

Comment:
There is no need for composer update here. 

See https://github.com/OskarStark/phpstan-ga/blob/master/entrypoint.sh#L8

Reply:
Hm... I was clearly wrong.. strange

Reply:
because of no-dev.

but looks like they added a new `$REQUIRE_DEV` variables.